src/Engine/Audio/SteamAudioRenderers.cpp

```diff
@@ -0,0 +1,118 @@
+#include "SteamAudioRenderers.hpp"
+
+SteamAudioRenderers::SteamAudioRenderers() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SteamAudioRenderers::convEffect' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)